pyhap/characteristic.py

```diff
@@ -216,3 +218,9 @@ def to_HAP(self):
             hap_rep['value'] = self.value
 
         return hap_rep
+
+    @classmethod
+    def from_dict(cls, name, json_dict):
```

Comment:
Should we add docs for the required fields?

Reply:
If you like me to, I will do them.

However personally I'm not a big fan of doc comments for obvious things. I don't remember where I've heard it, but someone once said: 'If you need comments for your code, then the code isn't good enough.'
In a broader context, I think the name of a function or method and its parameter should be enough to understand their meaning. For any method that is a direct way to interact with `HAP-python` I might understand it, but I believe it isn't necessary here. I would even go as far as to say that most of the comments for `TypeLoader`, `CharLoader` and `ServiceLoader` are unnecessary. Who ever needs to call these explicitly, probably can read python.

Reply:
True, but comments are used throughout and consistency always helps. My point was that a simple list with bullets like
- uuid
- HAP properties
    - format
is much less ambiguous and easy than to read than code. Anyway, I don't mind either way.